components/02-molecules/UserDropdown.tsx

```diff
@@ -19,9 +18,16 @@ export const UserDropdown = () => {
   const [avatarUrl, setAvatarUrl] = useState("");
   const [isLoading, setIsLoading] = useState(true);
 
+  const publicClient = usePublicClient();
+
   const router = useRouter();
 
   const getEnsAvatar = async () => {
+    if (!publicClient) {
+      toast.error("No public client found");
```

Comment:
```suggestion
      toast.error("No public client found. Please contact our team.");
```

I suggest this as visitors might not understand this language.